mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -889,16 +889,14 @@ trait Completions { this: MetalsGlobal =>
             param.name
         }
         .toSet
-      lazy val params: List[Symbol] = {
-        val prefix = ident.name.toString.stripSuffix(CURSOR)
-        baseParams.iterator
-          .filterNot { param =>
-            isNamed(param.name) ||
-            param.name.containsChar('$') // exclude synthetic parameters
-          }
-          .filter(param => param.name.startsWith(prefix))
-          .toList
+      val prefix = ident.name.toString.stripSuffix(CURSOR)
+      lazy val allParams: List[Symbol] = {
+        baseParams.iterator.filterNot { param =>
+          isNamed(param.name) ||
+          param.name.containsChar('$') // exclude synthetic parameters
+        }.toList
       }
+      val params = allParams.filter(param => param.name.startsWith(prefix))
```

Comment:
if this is a `val` then is there any gain from `allParams` being `lazy` ?

Reply:
Fixed it